pyramid/config/views.py

```diff
@@ -677,6 +673,22 @@ def wrapper(context, request):
           represents a header name or a header name/value pair, the
           case of the header name is not significant.
 
+        accept
```

Comment:
This was moved in https://github.com/Pylons/pyramid/pull/1487... I just apparently missed the narrative docs. Its not a proper predicate so you can't use `not_` on it.

Although `not_` really is a misfeature...

----

This is going to be confusing since for routes, it is still a predicate. *sigh*

Reply:
I updated the docs for both indicating which ones support it and which don't.